components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "IAM v2 Overview"
```

Comment:
think this was intended but missed

Reply:
@susanev you mean the `v2` should, or shouldn't be there?

Reply:
a previous pr docs team did, removed all the v2s so i meant that this one was prolly intended to be removed as well